src/camera.rs

```diff
@@ -32,6 +35,7 @@ impl CharacterControllerCameraOf {
             enable_smoothing: true,
             step_smooth_time: Duration::from_millis(200),
             teleport_detection_distance: 10.0,
+            yank_speed: 210.0_f32.to_radians(),
```

Comment:
Angle felt wrong for a value that gets aplied with delta seconds to me, if you really really want angle here I can change it though.